docs/sources/panels-visualizations/visualizations/alert-list/index.md

```diff
@@ -103,6 +103,10 @@ Filter alerts from the selected data source.
 
 Filter alerts by the selected folder. Only alerts from dashboards in this folder are displayed.
 
+### Show inactive
+
+Filter for Prometheus alerts with no instances. Prometheus alerts have zero (0) instances when in the Normal (inactive) state and these are hidden by default. You can choose to show them by toggling this switch. If the **Alert instance label** filter is configured, it takes precedence and inactive alert rules are hidden.
```

Comment:
Toggle to show alert rules with no alert instances. 

Alert rules have zero (0) alert instances when in **Normal**(inactive) state and are hidden by default.

Reply:
I am not sure what this means? If you have configured the **Alert instance label** filter, it takes precedence and inactive alert rules are hidden.

what does it take precedence over? If you've configured the filter then the toggle doesn't work?

Reply:
This comment above might explain the logic behind it a little more too - https://github.com/grafana/grafana/pull/96888#discussion_r1854630925

Ultimately I trialled both behaviors here (one where we ignore the instance label filter and this one, where we honor it). Considering the other filters again now, there is already some level of precedence which is implied; for example if someone filters by an alert rule name, and then also applies an instance label filter, the instance label filter will win.

In this case it might be over complicating things to explain that how it reacts to the Alert instance label filter?